toolbox/githubctl/main.go

```diff
@@ -120,6 +138,7 @@ func preprocessProwResults() map[string]map[string]bool {
 			}
 		}()
 	}
+	postSubmitJobs := postSubmitJobsMap[*baseBranch]
```

Comment:
you mgith want check that the key exist, and fail otherwise

Reply:
thanks seb. fixed it.